hoomd/md/EvaluatorPairTWF.h

```diff
@@ -0,0 +1,242 @@
+#ifndef __PAIR_EVALUATOR_TWF_H__
+#define __PAIR_EVALUATOR_TWF_H__
+
+#ifndef __HIPCC__
+#include <string>
+#endif
+
+#include "hoomd/HOOMDMath.h"
+
+/*! \file EvaluatorPairTWF.h
+    \brief Defines the pair evaluator class for TWF potentials
+    \details As the prototypical example of a MD pair potential, this also
+        serves as the primary documentation and base reference for the
+        implementation of pair evaluators.
+*/
+
+// need to declare these class methods with __device__ qualifiers when building
+// in nvcc
+// DEVICE is __host__ __device__ when included in nvcc and blank when included
+// into the host compiler
+#ifdef __HIPCC__
+#define DEVICE __device__
+#else
+#define DEVICE
+#endif
+
+
+//! Class for evaluating the TWF pair potential
+/*! <b>General Overview</b>
+
+    EvaluatorPairTWF is a low level computation class that computes the TWF pair
```

Comment:
I'd prefer to see documentation specific TWF documentation here rather than the generic docs copied from LJ.